test/format/README

```diff
@@ -0,0 +1,62 @@
+Tracing format operations can be useful in debugging.
+
+Turn on tracing with the "-t" flag:
+
+    -t	Trace operations into a WiredTiger log
+
+That causes format to trace operations in a WiredTiger log.
```

Comment:
Not sure who the audience is but when I read that I was thinking this was using `log_printf` into the actual WiredTiger log, i.e. `WiredTigerLog.000000#`.

Reply:
That's exactly what it does, it uses `log_printf` to insert messages into the WiredTiger log files.

I clarified a bit in 71ba2cef7.

Reply:
And that is exactly why I need to take another pass. I missed that! Thanks for the clarifications in the readme.